models/Document/Editable/Area.php

```diff
@@ -177,8 +194,9 @@ public function frontend(): void
 
         if ($dialogConfig) {
             $editableRenderer = \Pimcore::getContainer()->get(EditableRenderer::class);
+            $items = $this->renderDialogBoxEditables($dialogConfig->getItems(), $editableRenderer, $dialogConfig->getId());
+            $dialogConfig->setItems($items);
```

Comment:
Is this really required? as before rendering of the parent element was non-blocking.

Reply:
It's been quite a while, so I had a look again and it seems to be necessary, yes.

This is how it looks when updating the items _after_ the `outputEditmode` call _or_ keeping these lines as is:
<img width="715" alt="image" src="https://github.com/pimcore/pimcore/assets/57256630/db1978eb-c248-4264-8237-671028110006">

This is what it looks like when updating _before_ the `outputEditmode` call (aka. using the suggested changes):
<img width="613" alt="image" src="https://github.com/pimcore/pimcore/assets/57256630/7bc839c5-d366-43d8-86dd-e9b7ff9ed98b">

Please let me know if you have any suggestions on how to update the code :-)